test/morphology/test_black_hat.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+import torch
+import kornia.morphology.black_hat as black_hat
+import kornia.testing as utils  # test utils
+from torch.autograd import gradcheck
+
+
+class TestBlackHat:
```

Comment:
I miss some numeric unit tests. maybe you can generate some values from opencv

Reply:
+1 for adding numerical tests.

Reply:
I created a small test, but with custom numbers. That could be enough?